src/eckit/geo/iterator/Reduced.cc

```diff
@@ -0,0 +1,95 @@
+/*
+ * (C) Copyright 1996- ECMWF.
```

Comment:
Copyright date here and in every file should be up to date I guess.

Reply:
This is up to date, as far as I understand we decided to label these